lms/djangoapps/completion_api/tests/test_views.py

```diff
@@ -204,3 +206,60 @@ def test_staff_access(self):
         self.client.force_authenticate(user)
         response = self.client.get('/api/completion/v0/course/?user={}'.format(self.test_user.username))
         self.assertEqual(response.status_code, 200)
+
+
+class CompletionBlockUpdateViewTestCase(SharedModuleStoreTestCase):
+    """
+    Test that CompletionBlockUpdateView can be used to mark XBlocks as completed.
+
+    Ensure that it handles authorization as well.
+    """
+
+    usage_key = 'i4x://edX/toy/video/sample_video'
+
+    @classmethod
+    def setUpClass(cls):
+        super(CompletionBlockUpdateViewTestCase, cls).setUpClass()
+        cls.course = ToyCourseFactory.create()
+
+    def setUp(self):
+        super(CompletionBlockUpdateViewTestCase, self).setUp()
+        SignalDisconnectTestMixin.connect_signals()  # Disconnected by default in test settings.
```

Comment:
@bradmerlin We got a bunch of seemingly unrelated test failures but so far they do seem to only be happening on your branch :/ Any chance that this ^ line is having an effect on other tests in the test suite?